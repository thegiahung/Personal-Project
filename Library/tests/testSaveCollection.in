add book books.csv 111111
add book books.csv 111112
save collection books2.csv
add collection books.csv
save collection books3.csv
exit
