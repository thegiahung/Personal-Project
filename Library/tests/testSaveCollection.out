user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Success.

user: 15 books successfully added.

user: Success.

user: Ending Library process.
